Assets/Scripts/CombatManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CombatManager : MonoBehaviour
+{
+    public GameObject playerPrefab;
+    public GameObject enemyPrefab;
+
+    public CastButtonController castButton;
+
+    public int cardsRequired;
+
+    CreatureController target;
+
+    int cardsSelected;
+
+    void Start()
+    {
+        GenerateCombatParticipants();
+    }
+
+    void GenerateCombatParticipants() {
+        // Instantiate prefabs
+        GameObject player = GenerateCreature(playerPrefab);
+        GameObject enemy = GenerateCreature(enemyPrefab);
+    }
+
+    GameObject GenerateCreature(GameObject creaturePrefab) {
+        GameObject creature = Instantiate(creaturePrefab);
+
+        creature.transform.SetParent(gameObject.transform, false);
+
+        creature.GetComponent<CreatureController>().combatManager = this;
+
+        return creature;
+    }
```

Comment:
If this is starts with a `void GenerateCombatParticipants()` then why do the `GenerateCreature(GameObject creaturePrefab)` calls generate a GameObject reference?  What are we doing with the player and enemy references?  Can we just make this method void instead and just generate them?

Reply:
Was keeping track of them before, forgot to remove the references. Fixed.